lib/src/model.dart

```diff
@@ -1441,11 +1427,34 @@ abstract class GetterSetterCombo implements ModelElement {
     return allFeatures;
   }
 
-
   @override
   ModelElement enclosingElement;
   bool get isInherited;
 
+  String _constantValueBase;
+  String get constantValueBase {
+    if (_constantValueBase == null) {
+      if (element.computeNode() != null) {
+        var v = element.computeNode().toSource();
+        if (v == null) return null;
+        var string = v.substring(v.indexOf('=') + 1, v.length).trim();
+        _constantValueBase =
+            const HtmlEscape(HtmlEscapeMode.UNKNOWN).convert(string);
+      }
+    }
+    return _constantValueBase;
+  }
+
+  String linkifyWithModelType(String text) {
+    RegExp r = new RegExp("\\b${modelType.name}\\b");
```

Comment:
Is this called frequently? If so, we may want to cache the regexp in an instance field.

Reply:
This method is not called very often per GetterSetterCombo (maybe twice?).  Probably not worth it at this point.